src/components/chat/comment-card.scss

```diff
@@ -183,10 +202,43 @@ $user-icon-teacher-ada-border:       #949494;
       }
     }
 
+    .comment-agree {
+      position: absolute;
+      left: 4px;
+      top: 5px;
+      height: 48px;
+
+      .comment-agree-header {
+        font-style: italic;
+      }
+
+      .comment-agree-buttons {
+        display: flex;
+        flex-direction: row;
+
+        div {
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          margin: 4px 4px 0 0;
+          width: 56px;
+          height: 26px;
+          border-radius: 5px;
+          border: solid 1.5px $charcoal-light-1;
+          &:hover, &:active, &.selected {
+            background-color: $charcoal-light-4;
+          }
```

Comment:
It looks like "selected" is slightly darker than "hover" in the spec, not the same color.